userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -22,6 +22,28 @@ The full setup docs can be found [here](/docs/tutorial-basics/setup#prerequisite
 
 ## Simple install from a profile URL
 
+
+### Bootstrapping your local git repository
+You can use the `flux bootstrap` command to save commonly used `pctl` configuration to your GitOps repo.
```

Comment:
Shouldn't that be `pctl bootstrap`... ?

Reply:
lmao whoops

Reply:
see, this is what you get when you don't read my suggestions before merging. i hope you have learned your lesson.